src/routes/subscribe/addSubscriber.ts

```diff
@@ -0,0 +1,69 @@
+import { RequestHandler } from 'express';
+
+import responseTemplate from '../../utils/responseTemplate';
```

Comment:
You do not need to import and use this function. Just make sure that all responses are following ResponseTemplateType's structure. No import needed.